profiling/benchmark-cargo-test.sh

```diff
@@ -8,6 +8,10 @@ SERVER_PORT=8080
 PROFDIR=$(dirname "$0")
 ID=$(date +"%Y%h%d_%Hh%Mm%Ss")
 
+BRANCH_NAME=$(git symbolic-ref -q HEAD)
+BRANCH_NAME=${BRANCH_NAME##refs/heads/}
+BRANCH_NAME=${BRANCH_NAME:-HEAD}
+
```

Comment:
As `BRANCH_NAME` is `kai/benchmark_comparing`, the script fails because it tries to create a file under a subdirectory.
So I had to add a line here like `BRANCH_NAME=$(echo $BRANCH_NAME | sed -e 's/\//-/g')`

Reply:
thanks, good catch :)